docs/release_notes/ibexa_dxp_v5.0.md

```diff
@@ -10,6 +10,53 @@ month_change: true
 
 <div class="release-notes" markdown="1">
 
+[[% set version = 'v5.0.1' %]]
+
+[[= release_note_entry_begin("Ibexa DXP " + version, '2025-08-19', ['Headless', 'Experience', 'Commerce', 'New feature']) =]]
+
+#### Special characters in online editor
+
+The [online editor](online_editor_guide.md) now allows to easily enter special characters like currency symbols.
+It uses the [special characters plugin](https://ckeditor.com/docs/ckeditor5/latest/features/special-characters.html),
+
+![Special characters in online editor](4.6_special_characters.png "Special characters in online editor")
+
+#### Support for Solr 9
+
+With this release, [[= product_name =]] starts supporting [Solr 9](requirements.md#search).
+
+Solr 9 comes with support for [Dense Vector Search](https://solr.apache.org/guide/solr/latest/query-guide/dense-vector-search.html), paving the way for incoming improvements to the [AI Actions](ai_actions.md) feature.
+
+#### Improved content creation interface
+
+The editing interface of the back office has been improved to better highlight the language, creator, and the publication date when working with content items.
```

Comment:
We try to avoid Present Perfect Continuous, so we could change _has been improved_ to _is now improved_.

```suggestion
The editing interface of the back office is now improved to better highlight the language, creator, and the publication date when working with content items.
```